gulp-tasks/tests.js

```diff
@@ -8,6 +8,46 @@ var yargs = require('yargs');
 var buildFolder = path.join('dist', process.platform + '-' + process.arch);
 
 module.exports = function(gulp) {
+
+  function createMocha() {
+    return new mochaa({
```

Comment:
oh, thanks! I fixed that as different commit. This function is not needed, just leftover from my experiments.